### PR TITLE
Refactor documentation structure

### PR DESCRIPTION
**Pull request by oleksiidn**

### CODE REVIEW

File: information-mediator/docs/main.md [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+
+   and for the Security Server container (the same has to be done for each Security Server container if there are multiple):
+   ``` bash
+    $ docker cp information-mediator/scripts/security-server-entrypoint.sh <CID>/root/entrypoint.sh 
+   ```
+
+    * CID - docker container id (can be obtained by for example by ``` $ export CID=$(docker ps -aqf "name=cs") ``` where the  ```name``` is the container name
+
+   **NOTE**
+   The [central-server-entrypoint.sh](information-mediator/scripts/central-server-entrypoint.sh) and [security-server-entrypoint.sh](information-mediator/scripts/security-server-entrypoint.sh) scripts contain customizations to the original Docker entrypoint.sh script to configure the X-Road components to use remote database
```
Comment (bertvi) [inline information-mediator/docs/main.md:53]: These links do not work anymore as you have moved the documentation.

File: information-mediator/docs/main.md [hunk 2]
```diff
@@ -0,0 +1,473 @@
...
+Finally, that temporary values file is used to install IM to Kubernetes Cluster:
+``` bash
+ $ helm upgrade --install information-mediator ./information-mediator/ -f val.yaml 
+```
+
+11. Structure of Helm charts
+
+The "information-mediator" chart contains the following sub-charts:
+
+* A chart for [X-Road Central Server](information_mediator/charts/xroad-cs)
```
Comment (bertvi) [inline information-mediator/docs/main.md:411]: These links do not work anymore as you have moved the documentation.

File: information-mediator/docs/main.md [hunk 3]
```diff
@@ -0,0 +1,473 @@
...
+    - [secret.yaml](information_mediator/charts/xroad-ssc/templates/secret.yaml) for installing X-Road comnsumer Security Server secrets to Kubernetes Cluster
+* A chart for [X-Road provider Security Server](information_mediator/charts/xroad-ssp)
+  the sub-chart contains also the following templates:
+    - [application.yaml](information_mediator/charts/xroad-ssp/templates/application.yaml) for installing X-Road provider Security Server pod and service to Kubernetes Cluster
+    - [postgres.yaml](information_mediator/charts/xroad-ssp/templates/postgres.yaml) for installing X-Road provider Security Server PostgresSQL database to Kubernetes Cluster
+    - [secret.yaml](information_mediator/charts/xroad-ssp/templates/secret.yaml) for installing X-Road provider Security Server secrets to Kubernetes Cluster
+
+12. CI / CD pipeline
+
+There is also a [CircleCI pipeline](.circleci/config.yml) created for automation of those previous steps.
```
Comment (bertvi) [inline information-mediator/docs/main.md:434]: These links do not work anymore as you have moved the documentation.

**Review by bertvi — CHANGES_REQUESTED** [review]

File: information-mediator/docs/main.md (hunk 2 above)
Comment (oleksiidn) [reply information-mediator/docs/main.md]: Fix c84a4f112a5053772e60042ca260d90705e09962

File: information-mediator/docs/main.md (hunk 3 above)
Comment (oleksiidn) [reply information-mediator/docs/main.md]: Fix c84a4f112a5053772e60042ca260d90705e09962

File: information-mediator/docs/main.md (hunk 1 above)
Comment (oleksiidn) [reply information-mediator/docs/main.md]: Fix c84a4f112a5053772e60042ca260d90705e09962

**Review by bertvi — APPROVED** [review]

**Pull request merged**